addons/mail/models/mail_tracking_value.py

```diff
@@ -9,24 +9,22 @@
 class MailTracking(models.Model):
     _name = 'mail.tracking.value'
     _description = 'Mail Tracking Value'
-    _rec_name = 'field'
-    _order = 'tracking_sequence asc'
+    _rec_name = 'field_id'
+    _order = 'id DESC'
 
-    field = fields.Many2one('ir.model.fields', required=True, readonly=1, index=True, ondelete='cascade')
-    field_desc = fields.Char('Field Description', required=True, readonly=1)
-    field_type = fields.Char('Field Type')
+    field_id = fields.Many2one(
+        'ir.model.fields', required=True, readonly=True,
+        index=True, ondelete='cascade')
```

Comment:
So we still have the same issue as before.
When we remove a field, we lose all the tracking :/

Reply:
Ah indeed, but that's another feature. Here we mainly cleanup the model, then why not looking at this issue.

Reply:
The fact is that we may need to keep the `field_type` and `field_desc` fields to allow deletion (or simply field type change (from bool to selection, or from int to float)).

Reply:
Don't think we will store that much information just to support field deletion ... note that when removing a tracked field, tracking could be changed into a note, or something like that.